src/cryptext/session.py

```diff
@@ -51,6 +51,16 @@ def ensure_session(self, session_name: str) -> bool:
             session_name=session_name, ask_confirmation=True
         )
 
+    def plug_external(self, script_name: str = '') -> None:
+        """Take from external script one value for session name and list of PassordData"""
+        plugin = importlib.import_module(
```

Comment:
With the definition of a `PluginProtocol` (see the PR's discussion), you can add a type hint `plugin: PluginProtocol` to specify that the plugin contains certain attributes. This makes clear what should be inside the plugin, in particular the `imp_pass` list.

Reply:
In case other types of plug-ins are required, the `PluginProtocol` should be renamed in something more specific such as `PasswordPluginProtocol` but this is for now not required I think.